cmd/meta.go

```diff
@@ -17,6 +17,7 @@ func metaCommand() *cobra.Command {
 		RunE:  toRunE(metaMain),
 	}
 	metaCmd.Flags().StringP("output", "o", "json", "Set the output format (json or yaml)")
+	metaCmd.Flags().BoolP("attribute", "a", false, "Show the meta attribute instead of the full metadata")
```

Comment:
I think this could use a little more description. I'd add a Long description to `meta` that talks about the two ways we fetch metadata. I think we should also add a small note about it where `find` talks about `-fullmeta`.